templates/demo/template-primary-secondary.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta http-equiv="X-UA-Compatible" content="ie=edge">
+		<meta charset="UTF-8">
+		<link rel="stylesheet" href="../../components/demo/styles.css" type="text/css">
+		<script src="/node_modules/@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script type="module">
+			import '../../components/demo/demo-page.js';
+			import '../template-primary-secondary.js';
+		</script>
+	</head>
+	<body>
+		<d2l-demo-page page-title="Primary Secondary Template">

```

Comment:
The RTL toggle is a nice feature and feels like it would be important to test for this component, although hopefully grid will handle almost everything for us.  Should we also have an `index-rtl.html`?

Reply:
TBQH I think the RTL button can be misleading -- it sets `dir="rtl"` directly on the component, which can cause things to look fine even if the component is missing the `RtlMixin`.

For page templates, I think relying on a visual diff that does RTL testing is probably good enough, but yeah we could do an `index-rtl.html` if we wanted as well (although that duplicates a bunch of demo code).

Reply:
Oh _interesting_... you're right that is misleading, because I completely didn't think about that (even though it can only really be done that way, to work for each individual demo.  I wonder if it should be moved to just a higher-level toggle that would toggle the whole page and act more like a real-life scenario

Reply:
Ah, seems like this was already discussed here: https://github.com/BrightspaceUI/core/pull/289
Sorry, missed this!

Reply:
I've removed the `d2l-demo-page` and `d2l-demo-snippet` wrappers. Moved the demo folder into the `primary-secondary` folder and renamed this demo file to `index.html`.